kit/media/src/UIVideo/types.ts

```diff
@@ -11,6 +11,11 @@ export type UIVideoProps = {
      * `false` by default
      */
     controls?: boolean;
+    /**
+     * Determine whether the audio is muted
+     * `false` by default
+     */
+    muted?: boolean;
```

Comment:
I have already added this prop in my PR. Consider to remove to minimize conflicts.